src/constants/value.js

```diff
@@ -26,8 +26,8 @@ export const PERMISSION_QUESTIONS = {
 }
 
 export const MAX_COEFFICIENT = 100
-export const SMALL_COEFFICIENT = 25
-export const SUM_RATE_INDENTS = 38
+export const SMALL_COEFFICIENT = 30
```

Comment:
Did you mean `MIN_COEFFICIENT`?

Reply:
No. I mean small The coefficient can be less then 30. But when the coefficient is less than 30, percents did not  place in the block of the task rate